### DISCUSSION TITLE
Nginx Proxy Manager - access list

### DISCUSSION THREAD

**Discussion by gkalemis** [dani-garcia/vaultwarden#2716] [Q&A]
I am trying to block access to my vaultwarden server. I have internet access and SSL through Let's encrypt.

When I am adding basic access list through Nginx Proxy Manager I get the browser credential screen and after that the vaultwarden login screen but after clicking the Log-in the session is terminated and I am stuck at the login screen.

Any help will be awesome. Thank you.

![image](https://user-images.githubusercontent.com/24917402/187243772-0c275461-0c99-412e-82fc-7a15564db248.png)

![image](https://user-images.githubusercontent.com/24917402/187243876-b48feb24-2863-4e36-8d87-e0e7b55156a5.png)

![image](https://user-images.githubusercontent.com/24917402/187243506-42f6b183-6d69-44b5-9494-9ab849a498a8.png)

**Comment by BobWs** [comment]
Are you trying to completely block internet access to vaultwarden or do you want acces for specific users only?
In my opinion you could best use the NPM ACCES tab and allow only specific IP addresses and block everything else.

**Reply by gkalemis** [reply]
Thank you very much for the reply.  I have a dynamic IP.

**Reply by gkalemis** [reply]
And yes I want to block everyone except me from seeing the login page

**Reply by BobWs** [reply]
Take a look at this [#383](https://github.com/NginxProxyManager/nginx-proxy-manager/issues/383#issuecomment-1231453474) maybe you could try it.

**Comment by gkalemis** [comment]
Thanks but it is not working either.
